### ISSUE TITLE
Empty notes end up as non-empty HTML annotation

### ISSUE THREAD

**Issue by remram44** [VisTrails/VisTrails#1058]
An empty version note gives this HTML:
```html
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
<html><head><meta name="qrichtext" content="1" /><style type="text/css">
p, li { white-space: pre-wrap; }
</style></head><body style=" font-family:'MS Shell Dlg 2'; font-size:8.15094pt; font-weight:400; font-style:normal;">
<p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"><br /></p></body></html>
```

**Comment by dakoop** [comment]
As discussed in the meeting, this is an issue for upgrades because the "new" version may have this empty annotation that is viewed as the notes.

**Comment by remram44** [comment]
Fixed by 425224b3
